manifests/config/global.pp

```diff
@@ -16,6 +16,10 @@
   $split_key = split($name, '\.')
   $path = "/Users/${::boxen_user}/.gitconfig"
 
+  file { $path:
+    owner => $::boxen_user
```

Comment:
Are groups globally defined by a boxen project? I'm not sure what value we'd want to use for it otherwise.